README.md

```diff
@@ -66,15 +66,15 @@ This integration supports leaving inline PR comments via the Semgrep App. To lea
 
 ### Selecting Rules
 
-The `config` value lets you choose what rules and patterns semgrep should scan for.
-You can set specify rules in one of the following ways:
+The `config` values let you choose what rules and patterns semgrep should scan for.
```

Comment:
Is this enough of a change to make it clear that you can specify multiple configs?

Reply:
Looks good to me. You could also do something like this if you want to be more explicit:
```suggestion
The `config` values (one or more) let you choose what rules and patterns semgrep should scan for.
```